### DISCUSSION TITLE
How to solve this mess :-) ?

### DISCUSSION THREAD

**Discussion by mlerjen** [OpenOrienteering/mapper#2005] [Questions & Answers] [answered]
![image](https://user-images.githubusercontent.com/15611653/143835786-0a352684-7563-497f-b2f7-c0a9a0a806b2.png)
Hi,
I have a map, properly georeferenced but with the map origin way off the map. This works fine in Mapper but leads to a total misplacement (100km east and 50km north) for the OCAD export version.

My question now would be, how the proper workflow is in such a case. I just would like to get the map origin nearby the actual map without loosing the already perfect geo-reference.

Thank you for your advice!

**Comment by ollesmaps** [comment, marked as answer]
Perhaps an import to a predefined file with closer coordinates?

**Comment by mlerjen** [comment]
Yes.
My case somehow more complicated than it seemed. So after I :
1) opened a empty file
2) loaded a template ->geo-reference dialog
3) Imported the map
I got what I wanted.

But: opening in OCAD, OCAD still alerted that the map was outside the current drawing area and asked for a re-center. There is thus still some information form the previous map in the file, that influenced the map extent information.

I thus went back to Mapper and did a cutout on the actual map extent, with no success. The map extentis still to large.

In the end I could only solve the problem with OCAD->Map>Export part of map, which permanently remove the wrong map extent information.

??

**Reply by pkturner** [reply]
This use case is addressed by my "Georeferencing dialog ideas" prototype, #1702. It has a "Realign map" dialog where you can set new coordinates into the map reference point, shifting the map origin, without affecting the georeferenced positions of map objects.

**Reply by ollesmaps** [reply]
Just a wild guess: Is the template still opened when opening the file in Ocad? Closing it might help (or might not).
